components/KeyManagerPermissions/KeyManagerPermissions.tsx

```diff
@@ -39,20 +39,57 @@ const KeyManagerPermissions: React.FC = () => {
       <h2 className="title is-2">Permissions</h2>
       <article className="message is-info">
         <div className="message-body">
-          This tool will encode/decode permissions according to{' '}
-          <a href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager">
-            LSP6 KeyManager Standard
-          </a>{' '}
-          smart contract and attempt to match them with their{' '}
-          <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md">
+          This tool will encode and decode
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#address-permissions"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            permissions
+          </a>
+          based on the
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            LSP6 KeyManager
+          </a>
+          standardization to match them with their
+          <a
+            href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md"
```

Comment:
Let's update to

https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema